Fixes #3337 - Complete Overhaul of Newquay SMR

danielbutton opened the pull request:
Fixes #3337 

# Summary of changes

Oh God they let me loose with the pens again! As per the title, this is a complete re-do of this "SMR". As there is no actual SMR at Newquay, I have used mostly Mil colours for buildings, taxiways, etc. with a few Exeter colours added in to fill in the gaps where the Mil colours don't have a defined colour yet (mainly disused areas).

Changes include:

- Redraw of entire airport and perimeter
- Removal of unnecessary labels
- Addition of extra Geo to add taxiway centrelines and stand outlines
- Some nicer colours
- Removal of redundant double background
- Entirely new .kmz file for any future editors

# Screenshots (if necessary)

Old SMR
![NQY Old SMR](https://user-images.githubusercontent.com/46404477/103490454-bf8d8480-4e13-11eb-822a-44afa8a79434.png)

New SMR
![NQY New SMR Final NoCL](https://user-images.githubusercontent.com/46404477/104091030-a4d36980-5272-11eb-9295-73a02c872f6e.png)

Note: Purple background is changeable dependant on the "Active/Inactive Sector" colours as defined in EuroScope Symbology Settings, so this can/will change per profile. So for Area packs, the background will be grey (unless otherwise defined by the user).

PLM1995 commented:
Oooooh, nice!!! Looks much better then the existing imo. And changeable background colour is imo a good idea.

Two quick comments if you don't mind:

1. For me, the taxiway boundaries and centrelines and and hold lines around E and C look slightly cluttered/confusing at first glance/low zoom. I don't control EGHQ, so don't really care, but my take-it-or-leave-it suggestion would be to...
Either: Make the disused bits into a separate region of a different colour (e.g. 'smrTEDisused') and remove the boundary lines.
Or: Change the boundary line colour to something else (e.g. white) to indiciate these lines are not centre-lines.

2. I think in other SMR re-draws the old .kmz (in this case 'Newquay SMR.kmz') is kept for reference, and the new .kmz is added with an AIRAC date in its name. Might be an idea here? (Personally, I think saving .kml files to the repo would be more handy since they're probably more git-friendly?)

Nice SMR 🙂

danielbutton commented:
The stuff around Charlie and Echo was mainly for my own sanity as I have made this primarily for use in my own controller pack. When I was doing the Geo for this area, I drew the holding point lines as per Google Earth as I felt that drawing them all the way out to the edges looked silly. These extra bits then depict the defined width of the taxiway. More of a necessity for pilots than the controller, but still nice to have I think!

I originally had outlines for the Northern section of Charlie leading up to the Bristow apron, but I decided to drop this as it ended up looking like a really messy single line as the width was so low! I do see your point though, and this was one of my doubts about this project. I am inclined to change the outlines to "smrWhite" as you suggested just to aid in clarity. I'll post a side-by-side comparison below so we can review them against each other.

![NQY New SMR](https://user-images.githubusercontent.com/46404477/103492237-5e6cad80-4e21-11eb-9853-9943b2940512.png)

![NQY New SMR 2](https://user-images.githubusercontent.com/46404477/103492302-d20eba80-4e21-11eb-8d90-a8f6952266bb.png)

PLM1995 commented:
> I'll post a side-by-side comparison below so we can review them against each other.

Personally I do think the 'smrWhite' is clearer. But as I say I don't control EGHQ, so my opinion isn't particularly important :P Whichever you prefer! I was just making a suggestion in case you hadn't thought of it 🙂

danielbutton commented:
Oh yes, that isn't actually correct, I forgot to click "apply" when I loaded the SMR. The colour of the apron seems to have inverted, so now Aprons are using a random dark grey for some reason.

I was actually wondering if it would be a good idea to change the colour of the labels for the stands etc as they don't show up very well. If I can figure out which colour ES has chosen then I will define a new Apron colour to be used. Which colour did you use for your Southend SMR?

Anyway, I have updated the previous image to reflect the actual version (see above).

PLM1995 commented:
I think EGMC uses black currently, but the open PR uses 'smrGrey' as the darker grey and 'smrTEDisused' as the lighter grey. I think one of the heathrow colours (perhaps 'smrLLOutline' is also grey, but not sure how dark). I'm thinking I might need to define up to 3 new greys for the new style EGLL SMR, so maybe you could use those. I haven't worked out the exact colour yet, but was gonna sample the reference image we have to get them spot on.

danielbutton commented:
Oh right, part of me thought that PR had been merged already 😅 would've had a look myself if I'd have known that.

I'll have a play around tomorrow evening to see what I can come up with, although I'll most likely go with smrGrey if it's not too similar to the taxiway regions or disused areas.

For anyone else who reviews this, you may notice some slightly odd ordering of the Regions data. This is just so everything sits nicely in EuroScope (I was getting overhanging disused ramps on the Golf and Hotel aprons, one of the joys of working with ex-military airfields!)

PLM1995 commented:
Aye, you can have a play around and see what looks best 🙂 If you can't find enough/suitable shades of grey in the currently available ones, I've now sampled the ones I'm planning to add for LL, and they are as below. Note: I haven't yet tested these in ES, so hopefully I calculated them correctly.

| Colour | R-G-B | .sct val | Name |
| ----- | ----- | ----- | ----- |
| Runway background (light grey) | 90-90-90 | 5921370 | smrLLLightGrey |
| Disused areas (medium grey) | 60- 60- 60 | 3947580 | smrLLMidGrey |
| Grass/background (dark grey) | 50- 50- 50 | 3289650 | smrLLDarkGrey |

If I have got them correctly, they'll look like [this reference image](https://user-images.githubusercontent.com/13762210/103426117-fe6ed080-4bae-11eb-9469-bf5c8f9ec6bd.PNG).

danielbutton commented:
So after about half an hour of trying different things, I've finally settled on a suitable colour scheme. I'm now using smrGDapron for the aprons, a slightly darker shade than the smrMilApron. As for the taxiways, they've been redone using smrTETaxiway instead of smrMilTaxiway. Disused areas have also been darkened to make them more distinguishable from the taxiways.

![NQY New SMR Final](https://user-images.githubusercontent.com/46404477/103566705-60367f80-4eba-11eb-821b-087dc229605b.png)

danielbutton commented:
This will (hopefully) be the final update regarding this SMR. I have added a couple of final details, mainly in the Runway 30 Displaced Threshold area. I have also added an outline for the new width of the runway to make it look nicer. There are two extra labels now, one for the Air Ambulance landing pad and another for the Bristow SAR Apron.

![NQY New SMR Final CL](https://user-images.githubusercontent.com/46404477/104091123-42c73400-5273-11eb-9751-5edb62d74c7c.png)

Note: This screenshot was taken with the centreline option in EuroScope switched **on** and set to white with the "dash" line style. This quite nicely draws the centreline markings onto the SMR without adding to the size of the SMR itself.
